### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Silentsniper1** [UnamSanctam/SilentCryptoMiner#2204]
Can i use the same miner(Same Exe) to infect multiple devices or should I always create a new one can i copy paste the same exe over and over?

**Comment by UnamSanctam** [comment]
Yes you can run the same .exe on any amount of devices without issue.
